src/modules/app/StateType.js

```diff
@@ -71,6 +80,31 @@ export type EventRouteStateType = {|
   +message: ?string
 |}
 
+export type NewsRouteStateType = {|
+  +status: 'ready',
+  ...NewsRouteConfigType,
+  +models: $ReadOnlyArray<LocalNewsModel | TunewsModel>, // TODO
+  +allAvailableLanguages: $ReadOnlyMap<string, ?string> // including the current content language
+  |} | {|
```

Comment:
I know that status is used everywhere as text and we are following that to keep consistency 
but i would prefer to keep them as global constants

Reply:
This is how it's done across the code base.